homeassistant/components/enphase_envoy/number.py

```diff
@@ -89,6 +89,7 @@ async def async_setup_entry(
         envoy_data.tariff
         and envoy_data.tariff.storage_settings
         and coordinator.envoy.supported_features & SupportedFeatures.ENCHARGE
+        and coordinator.envoy.supported_features & SupportedFeatures.ENPOWER
```

Comment:
We could combine `SupportedFeatures.ENCHARGE | SupportedFeatures.ENPOWER` into a new constant

Reply:
Good suggestion for a future cleanup/PR, but not a real requirement IMHO for this PR to be blocked at this point.